web/src/engine/websites/InManga.ts

```diff
@@ -0,0 +1,96 @@
+import { Tags } from '../Tags';
+import icon from './InManga.webp';
+import { Chapter, DecoratableMangaScraper, Manga, Page, type MangaPlugin } from '../providers/MangaPlugin';
+import { FetchCSS, FetchJSON, FetchRequest } from '../FetchProvider';
+import * as Common from './decorators/Common';
+
+type APIChapters = {
+    data: string;
+}
+
+type APIChapter = {
+   result : {
+        Identification: string,
+        FriendlyChapterNumber: string
+    }[]
+}
+
+@Common.ImageDirect()
+
+export default class extends DecoratableMangaScraper {
+
+    public constructor() {
+        super('inmanga', `InManga`, 'https://inmanga.com', Tags.Language.Spanish, Tags.Media.Manga, Tags.Source.Aggregator);
+    }
+
+    public override get Icon() {
+        return icon;
+    }
+
+    private _createPostRequest(uri: string, form: string) {
+        const request = new FetchRequest(uri, { body: form, method: 'POST', referrer: this.URI.href });

```

Comment:
Is `referrer` supported, or should this be `headers: { referer: '' }`?

Reply:
referrer is indeed supported so i sued it. Which is kinda weird since its place is in headers.